WTWSMS/common/landed_titles/WtWSMS_zusk_titles.txt

```diff
@@ -302,16 +303,11 @@ e_khitan = {
 	allow = {
 		always = no # Only created through special event
 	}
+
+	male_names = { Aidi Aizong Bala Chengzong Chongzong Daizong Daozong Dezong Dingzong Duanzong Duzong Gaozong Gaozu Gongzong Guangzong Guizong Hendeji Houzhu Huaizong Huanzong Huizong Hula Jingzong Kangzong Liezu Lizong Mingzong Muzong Ningzong Qinzong Renzong Ruizong Shangdi Shaozhu Shenzong Shizong Shizu Shunzong Su Suzong Taizong Tianzuo Wa Wenzong Wuwang Wuzong Xiangzong Xianzhi Xianzhu Xianzong Xiaozong Xingzong Xizong Xuanzong Yelan Yingzong Yizong Yuanzong Zhangzong Zhaogu Zhaozong Zhengzong Zhenzong Zhongzhu Zhongzong Zhuangzong Zuwu }
 	
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
-	
-	# Regnal names	
-	male_names = {
-		Aidi Aizong Bala Chengzong Chongzong Daizong Daozong Dezong Dingzong Duanzong Duzong Gaozong Gaozu Gongzong Guangzong Guizong 
-		Hendeji Houzhu Huaizong Huanzong Huizong Hula Jingzong Kangzong Liezu Lizong Mingzong Muzong Ningzong Qinzong Renzong Ruizong 
-		Shangdi Shaozhu Shenzong Shizong Shizu Shunzong Su Suzong Taizong Tianzuo Wa Wenzong Wuwang Wuzong Xiangzong Xianzhi Xianzhu Xianzong Xiaozong 
-		Xingzong Xizong Xuanzong Yelan Yingzong Yizong Yuanzong Zhangzong Zhaogu Zhaozong Zhengzong Zhenzong Zhongzhu Zhongzong Zhuangzong Zuwu
-	}
```

Comment:
What is the point of the removal then? I don't get the nature of this change, so I'm always wary of encoding issues.